.env

```diff
@@ -27,6 +27,8 @@ SERPSTACK_API_KEY=#your serpstack api key here
 SEARCHAPI_KEY=#your searchapi api key here
 USE_LOCAL_WEBSEARCH=#set to true to parse google results yourself, overrides other API keys
 SEARXNG_QUERY_URL=# where '<query>' will be replaced with query keywords see https://docs.searxng.org/dev/search_api.html eg https://searxng.yourdomain.com/search?q=<query>&engines=duckduckgo,google&format=json
+BING_SEARCH_URL=#https://api.bing.microsoft.com/v7.0/search
```

Comment:
are there a lot of 3rd party search providers that use the bing API format? if it's only bing i'd prefer keeping only the `BING_SUBSCRIPTION_KEY` and hardcoding the `BING_SEARCH_URL` to avoid top-level env var clutter

Reply:
sure it is the same url for everyone i can remove that and hardcode it

Reply:
thanks!

Reply:
updated it